src/Components/Movies/Movies.js

```diff
@@ -2,13 +2,18 @@ import React from 'react';
 import './Movies.css'
 import MovieCard from '../MovieCard/MovieCard';
 
-const Movies = ({movies}) => {
+const Movies = ({ movies, seeMovieDetails}) => {
   const singleMovie = movies.movies.map(movie => {
     return (
-        <MovieCard image={movie['poster_path']}/>
-      )
+      <MovieCard
+        image={movie['poster_path']}
+        key={movie['id']}
+        id={movie['id']}
```

Comment:
I'd love to hear your reasoning behind choosing bracket notation over dot notation for the 3 lines above.